src/applications/hca/actions.js

```diff
@@ -29,22 +37,39 @@ export function getEnrollmentStatus(formData = {}) {
        action that corresponds to th condition you want to test:
        `dispatch({type: FETCH_ENROLLMENT_STATUS_FAILED, errors: [{ code: '404' }] });`
     */
+    if (simulateServer) {
```

Comment:
I got tired of manually editing the `getEnrollmentStatus` function so that it would work locally so I added this `simulateServer` flag that defaults to `true` if you are running the app locally.

Reply:
good idea- maybe create two functions and assign it based on the flag to `getEnrollmentStaus` before export. 

```
const getEnrollmentStatus = environment.isLocalHost() ? getTestEnrollmentStatus : getApiEnrollmentStatus

export getEnrollmentStatus
```